cohortextractor/patients.py

```diff
@@ -175,7 +175,11 @@ def registered_with_one_practice_between(
     return_expectations=None,
 ):
     """
-    All patients registered with the same practice through the given period
+    All patients registered with the same practice through the given period.
+
+    Note, it does not account for patients registered with one practice in a given time period, whose practice
+    changed its EHR provider during that that time period. To capture this information, please use
+    `with_complete_gp_consultation_history_between()`
```

Comment:
Insofar as possible, I'd suggest using the same language in the body of the docstring as in the title; writing in the present time ("simple present tense"); and preferring non-phrasal verbs (e.g. *return* rather than *account for*). For example:

> Note, this function does not return all patients registerd with the same practice through the given time period when this practice changes its EHR provider. To capture this information, please use `with_complete_gp_consultation_history_between()`.